fiftyone/core/video.py

```diff
@@ -273,16 +289,14 @@ def _sync_source_sample(self, sample):
         self._sync_source_schema()
 
         dst_dataset = self._source_collection._root_dataset
-        default_fields = set(
```

Comment:
nit: could removing the `set` have duplicate consequences 🤔

Reply:
In the new implementation, `_get_sample_only_fields()` directly returns set, so we're all set ✔️